### PR TITLE
style(examples): run prettier on the examples

### PR DESCRIPTION
**Pull request by rnike**
1. Add directory `examples` and file types `js` and `jsx` to the prettier script
2. Remove deprecated config `jsxBracketSameLine`
3. Create and add `/examples/build` to `.prettierignore` because we don't want to prettier the production code
4. `yarn prettier`

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/examples/build
```
Comment (dai-shi) [inline .prettierignore:1]: Can you configure to run prettier only for `examples/src` instead?

File: examples/package.json [hunk 2]
```diff
@@ -39,7 +39,6 @@
     "semi": false,
     "trailingComma": "es5",
     "singleQuote": true,
-    "jsxBracketSameLine": true,
```
Comment (dai-shi) [inline examples/package.json:42]: We have `"bracketSameLine": true,` in other projects. Could you add it?

File: package.json [hunk 3]
```diff
@@ -68,8 +68,8 @@
     "build:shallow": "rollup -c --config-shallow",
     "build:context": "rollup -c --config-context",
     "postbuild": "yarn copy",
-    "prettier": "prettier '*.{js,json,md}' '{src,tests,docs}/**/*.{ts,tsx,md,mdx}' --write",
-    "prettier:ci": "prettier '*.{js,json,md}' '{src,tests,docs}/**/*.{ts,tsx,md,mdx}' --list-different",
+    "prettier": "prettier '*.{js,json,md}' '{examples,src,tests,docs}/**/*.{js,jsx,ts,tsx,md,mdx}' --write",
+    "prettier:ci": "prettier '*.{js,json,md}' '{examples,src,tests,docs}/**/*.{js,jsx,ts,tsx,md,mdx}' --list-different",
     "eslint": "eslint --fix '*.{js,json}' '{src,tests}/**/*.{ts,tsx}'",
     "eslint:ci": "eslint '*.{js,json}' '{src,tests}/**/*.{ts,tsx}'",
```
Comment (dai-shi) [inline package.json:74]: I wonder if we can run eslint for `examples/src`. If it's not trivial, never mind.

**Review by dai-shi — CHANGES_REQUESTED** [review]
Thanks so much for working on this. A few suggestions:

File: package.json (hunk 3 above)
Comment (rnike) [reply package.json:74]: It does work by simply adding `examples/src` and `js,jsx` in the line, I'd like to look it through later and do this in another pull request.

File: examples/config-overrides.js [hunk 4]
```diff
@@ -1,6 +1,5 @@
 const { override } = require('customize-cra')
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin
 
-module.exports = override(
-  //addWebpackPlugin(new BundleAnalyzerPlugin())
-)
+module.exports = override()
+//addWebpackPlugin(new BundleAnalyzerPlugin())
```
Comment (dai-shi) [inline examples/config-overrides.js:5]: let's revert this one as it looks weird.
```suggestion
module.exports = override(
  //addWebpackPlugin(new BundleAnalyzerPlugin())
)
```

Comment (dai-shi) [reply examples/config-overrides.js:5]: btw, I'm fine to remove the plugin entirely. furthermore, remove customize-cra entirely.

File: package.json (hunk 3 above)
Comment (dai-shi) [reply package.json:74]: sounds good.

**Review by dai-shi — APPROVED** [review]

File: examples/config-overrides.js (hunk 4 above)
Comment (rnike) [reply examples/config-overrides.js]: Done.

I'm currently working on adding `@react-three/babel` which is still using `customize-cra`, this file on my branch is now like

```js
// examples/config-overrides.js
const { addWebpackPlugin, override, addBabelPlugin } = require('customize-cra')
const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin

const overrides = [
  addBabelPlugin('module:@react-three/babel'),
  process.env.BUNDLE_ANALYZE === 'true' ? addWebpackPlugin(new BundleAnalyzerPlugin()) : undefined,
]

module.exports = override(...overrides.filter((config) => config))
```

remove `customize-cra` then I'll need to add it back or have another approach, I'm leaving it there if no other reason to remove it.

Comment (dai-shi) [reply examples/config-overrides.js]: if you need it, no problem.

**Pull request merged**